notes/stv/20201009-Parquet-From-ObjectStore.txt

```diff
@@ -0,0 +1,107 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2019, ROE (http://www.roe.ac.uk/)
+#
+#     This information is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This information is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#  
+#     You should have received a copy of the GNU General Public License
+#     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#   </meta:licence>
+# </meta:header>
+#
+#
+
+
```

Comment:
It would be useful to reference how the cluster is configured.
If this is following on from previous notes, then just a one line comment referencing them would help to understand the context.
Are these steps being run on a master or worker node ?

Reply:
This was run on an automated deploy using the instructions from here: https://github.com/stvoutsin/aglais/blob/98b4e5746e53b4628e9f2d43d64b7f42d9eb91d1/notes/stv/20200915-automated-deploy-02.txt
These steps were run on a worker node, I will update the notes to make this clearer